Header header

int32 num_obj
float64[] obj_x
float64[] obj_y
float64[] obj_z